src/packaging/packages.targets

```diff
@@ -38,6 +38,8 @@
             <ILCompilerFiles Include="ILCompiler.MetadataTransform.dll" />
             <ILCompilerFiles Include="ILCompiler.MetadataWriter.dll" />
             <ILCompilerFiles Include="ILCompiler.TypeSystem.dll" />
+            <!-- This is needed for VS Code debugging for ILC - it only likes dll extension -->
```

Comment:
(And change the few places that do `corerun ilc.exe` to do `corerun ilc.dll` instead.)